rusoto/core/src/client_inner.rs

```diff
@@ -1,7 +1,7 @@
 use std::sync::Arc;
-use std::marker::PhantomData;
+use std::time::Duration;
```

Comment:
it'd be nice if we could make every import of duration the same. So we should either change this to: `use std::time::Duration as StdDuration`, or change the one in mock/src/lib.rs imo.

Reply:
I can change this one, no worries!